### PR TITLE
Migration to gold

### PR DESCRIPTION
**Pull request by Christopher-walsh22**
Testing works locally. Moving to dev for initial round.

### CODE REVIEW

**Review by davidclaveau — DISMISSED** [review]

**Comment by TimCsaky** [comment]
wondering if you need to update some of the SSO related values in the openshift templates in `api/openshift` and `app/openshift` and defaults in `api/.pipeline` and `app/.pipeline/`

File: .config/config.json [hunk 1]
```diff
@@ -37,14 +37,14 @@
             "realm": "standard"
         },
         "test": {
-            "url": "https://test.oidc.gov.bc.ca",
-            "clientId": "invasives-bc",
-            "realm": "dfmlcg7z"
+            "url": "https://test.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/certs",
```
Comment (davidclaveau) [inline .config/config.json]: Looks like additional paths added. I think these need to be just "https://test.loginproxy.gov.bc.ca"

File: .config/config.json [hunk 2]
```diff
@@ -37,14 +37,14 @@
...
+            "clientId": "inspect-bc-mussels-4817",
+            "realm": "standard"
         },
         "prod": {
-            "url": "https://oidc.gov.bc.ca",
-            "clientId": "invasives-bc",
-            "realm": "dfmlcg7z"
+            "url": "https://loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/certs",
```
Comment (davidclaveau) [inline .config/config.json]: Same as above, just "https://loginproxy.gov.bc.ca"

**Review by davidclaveau — CHANGES_REQUESTED** [review]
Just minor change to the KC urls in the config.json file

**Review by davidclaveau — DISMISSED** [review]

**Review by davidclaveau — DISMISSED** [review]

**Review by davidclaveau — APPROVED** [review]

**Pull request merged**